README_submit.md

```diff
@@ -34,9 +34,10 @@ submission with flavors
 
 ##### with local slave
 		
-	spark-submit --class "mytest.spark.CloudantApp" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.10/spark_test_2.10-0.1-SNAPSHOT.jar
-	spark-submit --class "mytest.spark.CloudantDF" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.10/spark_test_2.10-0.1-SNAPSHOT.jar
-	spark-submit --class "mytest.spark.CloudantDFOption" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.10/spark_test_2.10-0.1-SNAPSHOT.jar
+	spark-submit --class "mytest.spark.CloudantApp" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.11/spark_test_2.11-2.0.0.jar
+	spark-submit --class "mytest.spark.CloudantDF" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.11/spark_test_2.11-2.0.0.jar
+	spark-submit --class "mytest.spark.CloudantDFOption" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.11/spark_test_2.11-2.0.0.jar
+	spark-submit --class "mytest.spark.CloudantStreaming" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.11/spark_test_2.11-2.0.0.jar
```

Comment:
The 

`cloudant-spark.jar examples/scala/target/scala-2.11/spark_test_2.11-2.0.0.jar` 

did not work for me following the build instructions for scala examples. I had to point to a file with a name 

`examples/scala/target/scala-2.11/spark_test-assembly-2.0.0.jar` 

to get these 4 examples to run properly.

Is this a problem with either the build instructions or the execution instructions here? Does that work for you as prescribed?